bin/replication.js

```diff
@@ -48,6 +50,8 @@ function _createSetupReplication(command, options, log) {
             hosts: new RoundRobin(destination.bootstrapList[0].servers),
             transport: destination.transport,
         },
+        isTargetExternal,
```

Comment:
I suggest instead adding an `isExternal` option to the `target` config object because it's directly related to target configuration.

Reply:
Updated.